app/facades/item_facade.rb

```diff
@@ -0,0 +1,6 @@
+class ItemFacade
+  def self.item_info(id)
+    result = ItemService.get_item(id)[:data]
```

Comment:
I like that you're accessing `[:data]` and creating the Item object here, so that `ItemService` is only responsible for communicating with the API.